distributions/standalone/assembly-whitelist.txt

```diff
@@ -154,6 +154,7 @@
 /mule-standalone-${productVersion}/lib/mule/mule-transport-sockets-${productVersion}.jar
 /mule-standalone-${productVersion}/lib/mule/mule-module-tls-${productVersion}.jar
 /mule-standalone-${productVersion}/lib/mule/mule-module-tomcat-${productVersion}.jar
+/mule-standalone-${productVersion}/lib/mule/mule-module-tooling-${productVersion}.jar
```

Comment:
tooling support seems a better name. But this is a specialized service for tooling that should not be reused in the runtime. For instance to realize that there's a connectivity issue it may be that a mule component failed on initialization time. That would not be possible to be used in the runtime. And changing the model so it does not fail on initialization would require a major effort in the runtime.